jobs/README.md

```diff
@@ -7,6 +7,13 @@ commit, git push, and create a pull request on GitHub.
 __Be advised that Job Openings older than 3 months will be removed for the sake
 of keeping the board up to date. Feel free to repost.__
 
+## Frontend Developer (Remote) @ [Weave](https://weaveeducation.com/)
+Weave provides a centralized system for managing higher education assessment and accreditation that enables collaboration and helps create a campus-wide culture of continuous improvement.
+
+We are looking for a Frontend Developer to join our team. The position is remote for a US company, however we would prefer a person in Heraklion.
+
+[Read more & Apply
+here](https://apply.workable.com/weave-education/j/A0FC9EE467/)
```

Comment:
Can you please add a PubDate info as well in order to be able to keep track of job announcements? You can have a look on other jobs in the list about how we usually format this info.

Reply:
Sure thing, I just added it.